### PR TITLE
fix typos and codestyle

### PR DESCRIPTION
**Pull request by orgua**

### CODE REVIEW

**Comment by orgua** [comment]
additional commit with a run of [PowerShell-Beautifier](https://github.com/DTW-DanWard/PowerShell-Beautifier). Probably not wanted, but the more explicit commands have an advantage -> if unwanted you could just cherry-pick the typo-commit. 

I could also prepare a github action that test-runs the scripts in a W10 & W11 container and could function as a test-suite.

File: W10ANDW11-NETWORK-TCP-BACKSUBOPTIMIZATION.ps1 [hunk 1]
```diff
@@ -1,65 +1,65 @@
...
 Set-NetOffloadGlobalSetting -ReceiveSegmentCoalescing Enabled
 
 # SET PACKET COALESCING TO DEFAULT
 Set-NetOffloadGlobalSetting -PacketCoalescingFilter Enabled
 
 # SET TCP CONGESTION CONTROL TO DEFAULT
-netsh int tcp set supplemental template=Datacenter congestionprovider=CUBIC
-netsh int tcp set supplemental template=Datacentercustom congestionprovider=CUBIC
-netsh int tcp set global ECN=Disabled
+netsh int tcp Set-Variable supplemental template=Datacenter congestionprovider=CUBIC
```
Comment (kn0rki) [inline W10ANDW11-NETWORK-TCP-BACKSUBOPTIMIZATION.ps1]: netsh and set-variable? i think the powershell beautifier sets this?

Comment (orgua) [reply W10ANDW11-NETWORK-TCP-BACKSUBOPTIMIZATION.ps1]: good spot. yeah, the beautifier did this :( 
I reverted that commit

**Review by MysticFoxDE — APPROVED** [review]

File: W10ANDW11-NETWORK-TCP-BACKSUBOPTIMIZATION.ps1 (hunk 1 above)
Comment (MysticFoxDE) [reply W10ANDW11-NETWORK-TCP-BACKSUBOPTIMIZATION.ps1]: "netsh int tcp Set-Variable" ist wrong

**Review by MysticFoxDE — APPROVED** [review]

**Review by MysticFoxDE — APPROVED** [review]

**Review by MysticFoxDE — APPROVED** [review]

**Review by MysticFoxDE — APPROVED** [review]

**Review by MysticFoxDE — APPROVED** [review]

**Comment by Karl-WE** [comment]
I would like to add to this PR.

Please open the code in a proper rditor like Visual Studio Code instead of deprecated ISE @MysticFoxDE. 

You will see further improvement proposals on the bottom left tap and can fix them right click or search and replace

- replace aliases
- null parameters should be reversed

**Pull request merged**